addons/product/models/product_attribute.py

```diff
@@ -50,7 +50,10 @@ class ProductAttribute(models.Model):
         comodel_name='product.attribute.value',
         inverse_name='attribute_id',
         string="Values", copy=True)
-
+    template_value_ids = fields.One2many(
+        comodel_name='product.template.attribute.value',
+        inverse_name='attribute_id',
+        string="Template Values", copy=True)
```

Comment:
copy True ?
I don't think that the magic records linking templates and attribute values should be copied when the attribute is duplicated.

Reply:
Mybad, I will change this soon :-)